src/components/crossDevice/CrossDeviceSubmit/index.js

```diff
@@ -17,7 +17,10 @@ class CrossDeviceSubmit extends Component {
   }
 
   hasDocumentCaptureStep = () => {
-    return this.props.steps.some((step) => step.type === 'document')
+    return (
+      this.props.steps.some((step) => step.type === 'document') ||
+      this.props.mobileConfig.steps[0].type === 'document'
```

Comment:
why do we check only the type of the 1st step in the mobile config?

Reply:
This to be redundant by upcoming polling on crossDevice, however at this point there is always only 1 step in the list. that is why.